docs/transport/aviation/index.md

```diff
@@ -0,0 +1,193 @@
+
+ksqlDB supports tables and streams as objects. Both are backed by Kafka topics. Here we're going to create three tables in a normalised data model to hold information about our customers, their bookings, and the flights. 
+
+[//]: # "`TODO: Simple ERD of the three tables`"
+
+First off, let's create a table that will hold data about our customers: 
+
+```sql
+--8<-- "docs/transport/aviation/c01.sql"
+```
+
+This will store the data in a Kafka topic. In practice you would probably populate this directly from your application or a feed from your database using Kafka Connect. For simplicity, here we'll just load some data directly: 
+
+```sql
+insert into CUSTOMERS (id, name, address, email, phone, loyalty_status) values (1, 'Gleda Lealle', '93 Express Point', 'glealle0@senate.gov', '+351 831 301 6746', 'Silver');
+insert into CUSTOMERS (id, name, address, email, phone, loyalty_status) values (2, 'Gilly Crocombe', '332 Blaine Avenue', 'gcrocombe1@homestead.com', '+33 203 565 3736', 'Silver');
+insert into CUSTOMERS (id, name, address, email, phone, loyalty_status) values (3, 'Astrix Aspall', '56 Randy Place', 'aaspall2@ebay.co.uk', '+33 679 296 6645', 'Gold');
+insert into CUSTOMERS (id, name, address, email, phone, loyalty_status) values (4, 'Ker Omond', '23255 Tennessee Court', 'komond3@usnews.com', '+33 515 323 0170', 'Silver');
+insert into CUSTOMERS (id, name, address, email, phone, loyalty_status) values (5, 'Arline Synnott', '144 Ramsey Avenue', 'asynnott4@theatlantic.com', '+62 953 759 8885', 'Bronze');
+```
+
+Next we'll create a table of flights, and associated bookings for our customers. 
+
+
+```sql
+--8<-- "docs/transport/aviation/c02.sql"
+```
+
+For these two tables let's add some data. As before, this would usually come directly from your application or a stream of data from another system integrated through Kafka Connect. 
+
+```sql
+insert into flights (ID, ORIGIN, DESTINATION, CODE, SCHEDULED_DEP, SCHEDULED_ARR) values (1, 'LBA', 'AMS', '642',  '2021-11-18T06:04:00', '2021-11-18T06:48:00');
+insert into flights (ID, ORIGIN, DESTINATION, CODE, SCHEDULED_DEP, SCHEDULED_ARR) values (2, 'LBA', 'LHR', '9607', '2021-11-18T07:36:00', '2021-11-18T08:05:00');
+insert into flights (ID, ORIGIN, DESTINATION, CODE, SCHEDULED_DEP, SCHEDULED_ARR) values (3, 'AMS', 'TXL', '7968', '2021-11-18T08:11:00', '2021-11-18T10:41:00');
+insert into flights (ID, ORIGIN, DESTINATION, CODE, SCHEDULED_DEP, SCHEDULED_ARR) values (4, 'AMS', 'OSL', '496',  '2021-11-18T11:20:00', '2021-11-18T13:25:00');
+insert into flights (ID, ORIGIN, DESTINATION, CODE, SCHEDULED_DEP, SCHEDULED_ARR) values (5, 'LHR', 'JFK', '9230', '2021-11-18T10:36:00', '2021-11-18T19:07:00');
+```
+
+```sql
+insert into bookings (ID, CUSTOMER_ID, FLIGHT_ID) VALUES (1,2,1);
+insert into bookings (ID, CUSTOMER_ID, FLIGHT_ID) VALUES (2,1,1);
+insert into bookings (ID, CUSTOMER_ID, FLIGHT_ID) VALUES (3,5,3);
+insert into bookings (ID, CUSTOMER_ID, FLIGHT_ID) VALUES (4,4,2);
```

Comment:
Standard nitpick about SQL code-formatting. What's capitalised, what's not?:-)

Reply:
Good point. Fixed.